src/components/Home/Highlights/CustomFlatlist.tsx

```diff
@@ -0,0 +1,168 @@
+                                                fontWeight: "900",
+                                                color: "white",
+                                                fontSize: 20,
+                                                textAlign: "center",
+                                            }}
+                                        >
+                                            {formatTitle(item.title)}
+                                        </BodyText>
+                                    </View>
+                                </ImageBackground>
+                            </View>
+                            <View
+                                //this view represents the text under the highlights
+                                style={{
+                                    marginTop: 12,
+                                    width: width - 80,
+                                    justifyContent: "space-between",
+                                    alignSelf: "center",
+                                    flexDirection: "row",
+                                }}
+                            >
+                                <View>
+                                    <BodyText
+                                        style={{
+                                            fontWeight: "700",
+                                            marginLeft: 2,
+                                            fontSize: 12,
+                                        }}
+                                    >
+                                        {item.date}
+                                        {"          "}
+                                        {item.time}
+                                    </BodyText>
+                                </View>
+                                <View>
+                                    <BodyText
+                                        style={{
+                                            fontWeight: "400",
+                                            marginRight: 2,
+                                            fontSize: 12,
+                                        }}
+                                    >
+                                        {item.room}
+                                    </BodyText>
+                                </View>
+                            </View>
+                            <View
+                                //this view represents the line of separation
+                                style={{
+                                    width: width - 80,
+                                    marginTop: 12,
+                                    justifyContent: "center",
+                                    alignSelf: "center",
+                                    borderBottomColor: "#8791BD",
+                                    borderBottomWidth: 1,
+                                }}
+                            />
```

Comment:
Non che sia fondamentale, un po' di tempo fa avevo creato un componente apposta per fare la linea di separazione dato che compariva in diverse parti dell'app, se vuoi puoi usare questo.

```suggestion
                           <Divider
                                style={{ marginTop: 12, width: width - 80 }}
                            />
```

Reply:
top grazie, sostituisco subito

Reply:
cosi non funziona, ho aggiunto una prop "style" al componente Divider, dopo quando faccio il commit prova a vedere se va bene!Funzionare funziona